code/script.js

```diff
@@ -1,9 +1,27 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById('chat');
+const formSubmitButton = document.getElementById("send-btn");
+const formMessageBox = document.getElementById("name-input");
+const inputWrapper = document.getElementById("input-wrapper");
+
 
 // Global variables, if you need any, declared here
+let questionNumber = 1;
+let chatDelay = 800;
 
 // Functions declared here
+const botReply = (msg) => {
+  showMessage(msg, "bot");
+};
+
+const userReply = (msg) => {
+  showMessage(msg, "user");
+};
+
+const firstToUpperCase = (msg) => {
+  return msg.substr(0, 1).toUpperCase() + msg.substr(1);
+}
+
```

Comment:
Greate reusable functions 👍🏻 They make your code look cleaner and easier to read and understand 🦄